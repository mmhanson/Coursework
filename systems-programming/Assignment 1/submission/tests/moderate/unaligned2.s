main:
	movl	$4660, %ebx
	printr	%ebx
	movl	%ebx, 1(%eax)
	movl	0(%eax), %eax
	printr	%eax
