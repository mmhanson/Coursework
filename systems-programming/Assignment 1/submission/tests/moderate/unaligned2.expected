4660 (0x1234)
1192960 (0x123400)
